packages/xstate-persistence/src/types/types.ts

```diff
@@ -0,0 +1,35 @@
+import { GetStateArgs, IAbstractXStateStore, SaveStateArgs, State } from '@sphereon/ssi-sdk.data-store'
+import { IAgentContext } from '@veramo/core'
+
+import { IXStatePersistence } from './IXStatePersistence'
+
+export type XStateStateManagerOptions = { store: IAbstractXStateStore; eventTypes: Array<string> }
+
+export enum XStatePersistenceEventType {
+  EVERY = 'every',
+}
+
+export enum SQLDialect {
+  SQLite3 = 'SQLite3',
+  PostgreSQL = 'PostgreSQL',
+}
+
+export type DeleteExpiredStatesArgs = {
+  duration: number
+  dialect: SQLDialect
```

Comment:
This makes no sense to me. A caller should not have to know any implementation detail like this

Reply:
yeah, it didn't make any sense so changed the logic of deletion